example/resources/users.py

```diff
@@ -71,3 +71,13 @@ def replace(self, params):
             'href': new_item['photos'].url()
         }
         return user
+
+
+@royal.collection_config('users.followers')
+class UserFollowers(royal.Collection):
+    pass
+
+
+@royal.item_config('users.followers')
+class Follower(royal.Item):
+    pass
```

Comment:
@deuxpi I removed inheritance between Follower and Item. The goal here is to show declarative in action.
I'll come up with a clearer example.